pkg/asset/agent/manifests/nmstateconfig.go

```diff
@@ -80,8 +80,7 @@ func (n *NMStateConfig) Generate(dependencies asset.Parents) error {
 	installConfig := &agent.OptionalInstallConfig{}
 	dependencies.Get(agentHosts, installConfig, agentWorkflow, clusterInfo)
 
-	// Temporary skip in case of add nodes workflow. Main input for this
-	// asset will be retrieved from the nodes-config.yaml asset
+	// Not required for the add nodes workflow.
```

Comment:
I'm having trouble understanding why we wouldn't want the nmstateconfigs to be generated if they were provided in nodes-config.yaml.

Reply:
You were right @rwsu, it's effectively required. I've fixed it in the next PR: https://github.com/openshift/installer/pull/8093